modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -163,4 +167,16 @@ SearchContainer<AssetEntry> searchContainer = assetPublisherDisplayContext.getSe
 	if (assetEntryId) {
 		window.location.hash = assetEntryId;
 	}
-</aui:script>
\ No newline at end of file
+</aui:script>
+
+<c:if test="<%= Validator.isNotNull(friendlyURLChangedMessage) %>">
+	<aui:script>
+		Liferay.Util.openToast({
+			message: '<%= friendlyURLChangedMessage %>',
+			toastProps: {
+				autoClose: 20000,
```

Comment:
I think it is better to change it to 5000

Reply:
I think we should follow the time set in Web Content https://github.com/liferay/liferay-portal/blob/6221e884ae62e693f52fb358482185dd16383216/modules/apps/journal/journal-web/src/main/resources/META-INF/resources/friendly_url_changed_message.jspf#L42